react/fragments/orderForm.graphql

```diff
@@ -117,6 +117,10 @@ fragment OrderFormFragment on OrderForm {
     documentType
     phone
   }
+  clientPreferencesData {
+    locale
+    optinNewsLetter
```

Comment:
Sorry, I missed this in your `checkout-graphql` PR, but "Newsletter" is a single word. I think this should be named `optInNewsletter` instead.

Reply:
oh, ok! I'll open a PR in `checkout-graphql` then

Reply:
My bad 😞

Reply:
done vtex/checkout-graphql#55

Reply:
no problem @marcoskwkm!